### PR TITLE
43 ci generate release

### LINKED ISSUE #43: ci: generate release
The CI should build and generate a release each time a tag is pushed on the main branch.

### PR DESCRIPTION
**Pull request by Thechi2000**
Closes #43

### CODE REVIEW

**Review by LeanderCretegny — DISMISSED** [review]

**Comment by Thechi2000** [comment]
I added a small refactor of the CI, renaming workflows and files. I also added badges to the README to display the CI status of the main branch.

File: action.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: 'Hello World'
+description: 'Greet someone'
```
Comment (pmdlt) [inline action.yaml:2]: Maybe a better name ? :)

**Review by pmdlt — APPROVED** [review]

**Review by pmdlt — APPROVED** [review]
lgtm

**Review by quartztz — APPROVED** [review]
looks great ! can be merged

**Pull request merged**